README.md

```diff
@@ -25,8 +28,24 @@ const Hello = App.State<string>('the sound of music');
 
 Pulse provides a clean-cut toolset to build a Javascript application quickly and efficiently. It encourges developers to construct a core library that can be dropped into any UI framework. Your `core` is the brain of your application, it will handle everything from state management, API requests to all logic and calculations. Pulse will supply pre-computed data to your UI components, in the framework of your choice with complete reactivity.
 
-### Typescript
+### TypeScript
+
+Pulse is written in TypeScript and is designed to support it heavily. Everything is type safe **out of the box** and encourages you to write clean, fully-typed code.
+
+### Core
+
+Pulse applications are managed and mostly centered around the `core` object. The core is a single object that contains all Controllers, State, Collections, actions, routes, and helpers as you define them under a Pulse instance.
 
-Pulse is written in Typescript and is designed to support it heavily. Everything is type safe out of the box and encourages you to write clean typed code.
+Think of the core as your custom library of state and methods in a portable container that can be deployed in any JS environment or framework.
+
+It's the final product of a Pulse powered application.
+
+```ts
+core.accounts.login();
+
+core.posts.collection.findById(21);
+
+core.authentication.state.TOKEN.value;
+```
```

Comment:
This is a section that's better to keep on the docs site itself. The readme is meant to be a small brief, but is not updated in line with the full docs, which will cause it to fall out of line with the current Pulse version and possibly cause confusion.